src/Runner.Worker/ExecutionContext.cs

```diff
@@ -105,6 +105,8 @@ public interface IExecutionContext : IRunnerService
         // others
         void ForceTaskComplete();
         void RegisterPostJobStep(IStep step);
+        IStep RegisterCompositeStep(IStep step, DictionaryContextData inputsData);
+        void EnqueueAllCompositeSteps(Queue<IStep> steps);
```

Comment:
Then we can eliminate this function? (switch `JobSteps` to List instead of Queue)